teamsupport/__init__.py

```diff
@@ -1,11 +1,17 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
+import teamsupport.config as config
 from teamsupport.models import Action, Ticket, User
 from teamsupport.services import TeamSupportService
 
 __author__ = 'Yola Engineers'
 __email__ = 'engineers@yola.com'
-__version__ = '0.1.3'
+__version__ = '0.1.4'
 
-__all__ = (Action, TeamSupportService, Ticket, User)
+__all__ = (Action, TeamSupportService, Ticket, User, config)
+
+
+def init(org_id, auth_key):
```

Comment:
As for me, `init` isn't worse than `configure`. It initializes library. I don't want  user to know that I log in to TeamSupport service inside - this looks like violation of encapsulation.